src/breakupModel/input/TLEReader.cpp

```diff
@@ -1,36 +1,69 @@
 #include "TLEReader.h"
 
+const std::map<char, size_t> TLEReader::alpha5NumberingSchemeOffset{
+        {' ', 0},
+        {'0', 0},
+        {'1', 10000},
+        {'2', 20000},
+        {'3', 30000},
+        {'4', 40000},
+        {'5', 50000},
+        {'6', 60000},
+        {'7', 70000},
+        {'8', 80000},
+        {'9', 90000},
+        {'A', 100000},
+        {'B', 110000},
+        {'C', 120000},
+        {'D', 130000},
+        {'E', 140000},
+        {'F', 150000},
+        {'G', 160000},
+        {'H', 170000},
+        {'J', 180000},
+        {'K', 190000},
+        {'L', 200000},
+        {'M', 210000},
+        {'N', 220000},
+        {'P', 230000},
+        {'Q', 240000},
+        {'R', 250000},
+        {'S', 260000},
+        {'T', 270000},
+        {'U', 280000},
+        {'V', 290000},
+        {'W', 300000},
+        {'X', 310000},
+        {'Y', 320000},
+        {'Z', 330000}
+};
 
 std::map<size_t, OrbitalElements> TLEReader::getMappingIDOrbitalElements() const {
     std::map<size_t, OrbitalElements> mapping{};
 
     std::ifstream fileStream{_filepath};
+    std::string line2;
 
-    while (!fileStream.eof()) {
-        auto pair = getTwoLine(fileStream);
-        mapping.insert(pair);
+    while (std::getline(fileStream, line2)) {
```

Comment:
does it make sense to throw some warning / exception if the line is never found?

Reply:
I would not throw any exception here, because if the TLEReader never finds a line 2, the mapping will be empty (and the user will notice this quickly).
Nevertheless, i agree that an exception could improve error handling/ finding. Maybe i will locate that in the SatcatTLEReader and the YAMLReader which both uses the TLEReader